### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justafish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,18 +33,15 @@ composer run-script install
 
 ## Running
 ```
-cd docroot
```
Comment (dawehner) [inline README.md]: This is a regression of https://github.com/jsdrupal/drupal-admin-ui/pull/107

**Comment by justafish** [comment]
https://github.com/jsdrupal/drupal-admin-ui-demo/pull/17 as a potential solution

**Comment by dawehner** [comment]
Let's get https://github.com/jsdrupal/drupal-admin-ui/pull/110 and https://github.com/jsdrupal/drupal-admin-ui-demo/pull/16 in first otherwise this documentation is a bit wrong.

File: README.md [hunk 2]
```diff
@@ -32,24 +33,21 @@ composer run-script install
...
 ```
 
 Try visiting one of the converted pages, e.g. the user permissions or roles page.
 
 ## Developing
 
 - Ensure you have [Node 8](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/) installed.
-- Make sure the webserver is started with `composer run-script start`
-- As we run the development environment as anonymous user right now, you need to grant them permissions.
-- Please do that in the "production" environment, like ```http://localhost:8888/admin/people/permissions```. The permissions you want to grant at least are "administer users" and "Administer site configuration".
```
Comment (dawehner) [inline README.md]: This is much nicer now we actually understand how it works :)

**Pull request merged**